trezorlib/tests/device_tests/test_msg_cardano_sign_message.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2018 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from .common import TrezorTest
+from binascii import hexlify
+from trezorlib.cardano import sign_message
+from trezorlib.tools import parse_path
+
+
+@pytest.mark.cardano
+@pytest.mark.skip_t1  # T1 support is not planned
+class TestMsgCardanoGetPublicKey(TrezorTest):
+
+    def test_cardano_get_public_key(self):
```

Comment:
also this

while you're at it, consider using `@pytest.mark.parametrize` to simplify test code here, see e.g. `test_cancel`